### ISSUE TITLE
Suggestions - Customize characters and positions; Use the numbers row to enter characters

### ISSUE THREAD

**Issue by jgfmachado** [damienleroy/PowerAccent#13]
Amazing work, it amazes me how just a few days ago I was thinking why Windows doesn't have this feature, similar to the Mobile keyboards, and then I receive an update on PowerToys 🤯.

Anyway, here are my suggestions:
- **Customize** our own available **special characters and its positions**.
- **Use the numbers row** on the keyboard, to immediately **enter the special character on the caret position**.
   - ie. Let's take the letter 'e' caret that currently contains [é][è][ê][ë][ē][ė][€]. If the user presses 'e'>'3'**(*)** it would immediately enter an [ê].
   **(*)** EDIT: to make this more versatile, we could allow any combination for the desired character, ie. 'e'+'9'='é'

**Comment by damienleroy** [comment]
I think that is a great idea!

Need to think about some things (depending of the keyboards, some shortcut can be really different).
But I will keep that in mind. Thanks :)

**Comment by stefanfritsch** [comment]
Not just mathematical symbols. E.g. it would fix the problem of needing a 105 key layout with a 104 key keyboard - just distribute the symbols of key 105 among the others.

### Plus:

If you're a programmer with a US layout seemingly half of what you type is done with your right pinky while in other languages you have to use AltGr a lot. You could map all the usual (braces, brackets, slashes, etc.) to your other fingers and be faster even though you have to type key combinations. 

But, you'd need an activation key that is less often used than `space`. 

Currently for any activation time less than 200ms I get too many mistypes. But that wouldn't be an issue, e.g. for the "w" key. I'd wager that braces and brackets on j+w, k+w, l+w and ;+w with a 50ms activation would be a good thing for a lot of people's tendons.

It seems like it should be possible to reuse a lot of the necessary UI from the keyboard manager.

**Comment by dj3underscores** [comment]
Exactly. The ability to add the characters which are necessary to me personally would be just perfect, as well as ordering the characters manually the way which is convenient to me.
